src/tt.cpp

```diff
@@ -30,6 +30,9 @@
 
 namespace Stockfish {
 
+// Per commit f7b3f0e, we use `bool(depth8)` to test if an entry is occupied. However we also need to
```

Comment:
i find linking to a specific commit a bit weird

Reply:
Yes, on the other hand it does contain a lot more relevant detail. I thought best to keep comments short but leave a pointer to more should the reader be interested...

do you have alternatives in mind? not sure how best to do this

Reply:
mh okay just had a look at the pr and it seems it had a bit more detail than the avg one so can stay, just dont want all comments to be cluttered with "got introduced with xyz"

Reply:
perhaps it should say "per the details in commit f7b3f0e"?

Reply:
though what is the comment doing at the save function ? it aint being tested there only in the probe, where the meaning is kinda self explanatory `found = bool(tte[i].depth8)` could point to the commit there

Reply:
well i put the comment where we make heaviest use of DEPTH_ENTRY_OFFSET, since the comment is ultimately about that constant and why it exists. im happy to move it where you like

Reply:
On further thought, based on your comments, I reviewed the old commit and concluded that more of its message should have been code-comments to begin with, so added those and removed the direct reference to the commit.

Is the resulting comment sufficiently short?